include/Song.h

```diff
@@ -101,14 +101,22 @@ class EXPORT Song : public TrackContainer
 	{
 		return m_nLoadingTrack;
 	}
+
 	inline int getMilliseconds() const
 	{
 		return m_elapsedMilliSeconds;
 	}
-	inline void setMilliSeconds( float ellapsedMilliSeconds )
```

Comment:
@PhysSong Yes, we can always add it back if needed. However, for now it look like the milliseconds are rather an internal representation of the `Song` class and that other client "think" in different times, e.g. `MidiTime`.